unicorn_mode/samples/simple/simple_test_harness.py

```diff
@@ -120,6 +102,8 @@ def main():
 
     uc.mem_map(STACK_ADDRESS, STACK_SIZE)
     uc.reg_write(UC_MIPS_REG_SP, STACK_ADDRESS + STACK_SIZE)
+    
+    print(STACK_ADDRESS + STACK_SIZE)
```

Comment:
is that a debug print that is still in there?

Reply:
Removed :)